src/DwitTech.AccountService.Data/Entities/UserLogOn.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DwitTech.AccountService.Data.Entities
+{
+    public class UserLogOn:BaseEntity
+    {
+        public int UserId { get; set; }
+        public string RefreshToken{ get; set; }
+        public DateTime RefreshTokenExpiryTime { get; set; }
```

Comment:
Why dont we use a envirinment variable to handle expiry time frame and check if the created or last updated time + expiry time frame has passed.

Reply:
That makes sense. I'll take out the RefreshTokenExpiryTime then.

Reply:
Just to be clarify, this validation will be done at controller level by whoever is handling that right?